number_parser/parser.py

```diff
@@ -109,16 +131,16 @@ def parse_number(input_string, language='en'):
         return int(input_string)
 
     tokens = _tokenize(input_string)
-
+    lower_case_tokens = []
     for index, token in enumerate(tokens):
         compare_token = token.lower()
+        lower_case_tokens.append(compare_token)
         if compare_token in lang_data.all_numbers or compare_token.isspace() or len(compare_token) == 0:
             continue
         if compare_token in lang_data.skip_tokens and index != 0:
             continue
         return None
```

Comment:
I would do:
```python
tokens = _tokenize(input_string)
normalized_tokens = _normalize_tokens(tokens)

for index, token in enumerate(normalized_tokens):
...
```

And extract the "`lower` logic" inside `_normalize`. We will probably need that function in the future to implement the Unicode normalization, so we can create it.

Tip: using `list(map(str.lower, tokens))`

Another way would be applying the `lower()`/normalization to the whole `input_string`, but I didn't try it and I don't know the consequences.